test/integration/test_kongplugininstallation.go

```diff
@@ -212,7 +207,9 @@ func TestKongPluginInstallationEssentials(t *testing.T) {
 func deployGatewayWithKPI(
 	t *testing.T, cleaner *clusters.Cleaner, namespace string,
 ) (gatewayIPAddress string, gatewayConfigNN, httpRouteNN k8stypes.NamespacedName) {
-	gatewayConfig := helpers.GenerateGatewayConfiguration(namespace)
+	// NOTE: Disable webhook for KIC, because it checks for the plugin in Kong Gateway and rejects,
+	// thus it requires strict order of deployment.
```

Comment:
```suggestion
	// NOTE: Disable webhook for KIC, because it checks for the plugin in Kong Gateway and rejects,
	// thus it requires strict order of deployment which is not guaranteed.
```